films_imdb/films_functions_spark/function_films/function_imdb1.py

```diff
@@ -0,0 +1,63 @@
+from pyspark.sql import SparkSession
+from pyspark import SparkConf
+import pyspark.sql.types as t
+import pyspark.sql.functions as f
+
+
+spark = SparkSession.builder\
```

Comment:
for long lines of code with method chaining, it's better to use parentheses `(...)`, instead of  slash `\`, since it's "more"  python-way
e.g.
use:
```python
var = (some().very_long().chained()
       .method_line().sort())
```
instead of:
```python
var = some().very_long().chained() \
    .method_line().sort()
```